components/ws-daemon/pkg/content/service.go

```diff
@@ -743,15 +743,7 @@ func (s *WorkspaceService) WaitForInit(ctx context.Context, req *api.WaitForInit
 	//nolint:ineffassign
 	span, ctx := opentracing.StartSpanFromContext(ctx, "WaitForInit")
 	tracing.ApplyOWI(span, log.OWI("", "", req.Id))
-	defer func() {
-		// https://github.com/gitpod-io/gitpod/issues/11713
-		if err != nil && strings.Contains(err.Error(), "cannot find workspace") {
-			log.WithFields(log.OWI("", "", req.Id)).Warn(err)
-			tracing.FinishSpan(span, nil)
-		} else {
-			tracing.FinishSpan(span, &err)
-		}
-	}()
+	tracing.FinishSpan(span, &err)
```

Comment:
👋 was scanning some PRs to build up context and noticed this, should `FinishSpan` stay `defer`ed? Same below

Reply:
Oops. You are :100: right. Good catch!